spec/base_spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 
 # for full configuration, see .simplecov
 require 'simplecov' if ENV["COVERAGE"]
+require 'simplecov-lcov' if ENV["COVERAGE"]
+require 'undercover' if ENV["COVERAGE"]
```

Comment:
We don't need these anymore because the LCOV conversion happens in a rake task, not when running Rspec.